python/spacetime/store_server.py

```diff
@@ -90,6 +100,8 @@ def get(self, sim):
         ret["new"] = all_new
         ret["updated"] = all_updated
         ret["deleted"] = all_deleted
+
+        reset_timer(sim)
```

Comment:
If a call takes more than 30 secs there is something wrong with it :) but yea. I think you should disable and reenable I think. Makes more sense.